### PR TITLE
Add VeePN.app v2.0.1

### PR DESCRIPTION
**Pull request by box000**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Welcome. Looking good so far. Made a few requests.

File: Casks/veepn.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask "veepn" do
+  version "2.0.1"
+  sha256 "6aa503a97f9520d6047159299cf4d97e56fcce68ca85deaf1ad9e14c48869a7c"
+
+  url "https://download.veepn.com/mac/VeePN_v#{version}.dmg"
+  appcast "https://download.veepn.com/mac/appcast.xml"
+  name "VeePN"
+  desc "VPN client for secure internet access and private browsing"
```
[the remark this answers is not in the document]
Comment (box000) [reply Casks/veepn.rb]: +

File: Casks/veepn.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  homepage "https://veepn.com/vpn-apps/vpn-for-mac/"
+
+  app "VeePN.app"
+
+  uninstall launchctl: "com.veepn.macos.helper"
```
[the remark this answers is not in the document]
Comment (box000) [reply Casks/veepn.rb:17]: +

**Review by vitorgalvao — DISMISSED** [review]

**Comment by vitorgalvao** [comment]
Great! Thank you.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**